<launch>

  <arg name="rviz" default="true" />

  <node pkg="covgrid_slam" type="scanRegistration" name="scanRegistration" output="screen">
      <remap from="/velodyne_points" to="/velodyne_points" />
      <remap from="/imu/data" to="/raw_imu" />  
      <!-- <remap from="/imu/data" to="/djiros/imu" />   -->
  </node>

  <node pkg="covgrid_slam" type="laserOdometry" name="laserOdometry" output="screen" respawn="true"/>



  <node pkg="covgrid_slam" type="laserMapping" name="laserMapping" output="screen"/>
  <node pkg="covgrid_slam" type="transformMaintenance" name="transformMaintenance" output="screen"/>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find covgrid_slam)/rviz_cfg/covgrid_slam.rviz" />
  </group>

</launch>
